dace/sdfg/performance_evaluation/op_in_helpers.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2019-2023 ETH Zurich and the DaCe authors. All rights reserved.
+""" Contains class CacheLineTracker which keeps track of all arrays of an SDFG and their cache line position
+and class AccessStack which which corresponds to the stack used to compute the stack distance. 
+Further, provides a curve fitting method and plotting function. """
+
+from dace.data import Array
+import sympy as sp
+from collections import deque
+from scipy.optimize import curve_fit
```

Comment:
We should wrap this in case the package is not installed. Scipy is not a dependency of DaCe, and is only used for something optional here, so we should check if it's present and if not print a warning that states that this analysis requires scipy to work.